### DISCUSSION TITLE
Script error after update

### DISCUSSION THREAD

**Discussion by gi-b716** [ORelio/Minecraft-Console-Client#2753] [Q&A] [answered]
I wrote a [script](https://gist.github.com/gi-b716/97a8939d34f5924e31872bfe8cc7ca90) when I was using version 20240415-263. Now I updated to 20240703-271, and the script gave the following error:
[![pkRfz28.png](https://s21.ax1x.com/2024/07/05/pkRfz28.png)](https://imgse.com/i/pkRfz28)
(Starting from line 16)
I can see that Centry is mentioned in the error. I looked through the commit records and found that Sentry was introduced in [this commit](https://github.com/MCCTeam/Minecraft-Console-Client/compare/20240415-263...20240703-271#diff-79c4148fc2039cff2f0709adc7a9e3a9d0697016d13589479f7de7e45a016057). But I think it has nothing to do with my script. What should I do?

**Comment by breadbyte** [comment]
Can you clear your temp folder and try running the script again? I cannot replicate the issue with your script.

**Comment by breadbyte** [comment, marked as answer]
This has been fixed in the latest update. Please update MCC.
